src/io/flutter/preview/PreviewView.java

```diff
@@ -545,13 +556,8 @@ public void customizeCellRenderer(
       setIcon(icon);
     }
 
-    // Render the parent/child association.
```

Comment:
You're right, most associations are just `child:` (I think we don't even show `children:` exactly because it is not interesting for `Row` or `Column`). But some widgets might have interesting associations, like `Scaffold` (`appBar` or `body`) or `AppBar` (`title` or `bottom`). In WindowBuilder we would display these "places" in the components tree to allow uses to drop new components there. We don't have any DnD yet, so maybe not very useful. We can discuss tomorrow.